tests/snuba/api/endpoints/test_organization_events_spans_histogram.py

```diff
@@ -0,0 +1,33 @@
+from django.urls import reverse
+
+from sentry.testutils.cases import APITestCase, SnubaTestCase
+
+
+class OrganizationEventsSpansHistogramEndpointTest(APITestCase, SnubaTestCase):
+    FEATURES = ["organizations:performance-span-histogram-view"]
+    URL = "sentry-api-0-organization-events-spans-histogram"
+
+    def setUp(self):
+        super().setUp()
+        self.features = {}
+        user = self.create_user()
+        self.login_as(user=user)
+        self.org = self.create_organization(owner=user)
+
+    def test_no_feature(self):
+        url = reverse(
+            self.URL,
+            kwargs={"organization_slug": self.org.slug},
+        )
+        response = self.client.get(url, format="json")
+        assert response.status_code == 404, response.content
+
+    def test_endpoint(self):
```

Comment:
We should probably modify this a bit, or add one more test that covers querying actual data, you'll want to copy / refactor the `create_event` function (https://github.com/getsentry/sentry/blob/master/tests/snuba/api/endpoints/test_organization_events_spans_performance.py#L57) from the spans tests. @Zylphrex will probably have better insight on which one to re-use though.

Reply:
You'll want something like the `create_event` from the other spans endpoint tests. You can use it as a reference but I wouldn't copy it exactly because the use case here is for histograms so you'll want data to test the histogram specifically. You'll likely want a helper function that lets you configure what the histogram will look like.